mix/ClientModel.cpp

```diff
@@ -111,7 +111,7 @@ QString ClientModel::apiCall(QString const& _message)
 
 void ClientModel::mine()
 {
-	if (m_running || m_mining)
```

Comment:
because now it is possible to mine when running a bunch of transactions (execution can contains multiple blocks).